.storybook/config.js

```diff
@@ -19,6 +19,12 @@ const stories = [
   // Welcome
   require.context("./", false, /welcome.story.js/),
 
+  // Toast
```

Comment:
Why can't this just say `require.context("../packages", true, /\.stories\.(js|mdx)$/),`?

Reply:
This is our way to sort it on the side nav :S cc @mikrotron

Reply:
Definitely still experimental + unsatisfactory, but I'm very interested in being able to to define the ordering for stories.  For now, alphabetic would be better than having these _two_ out of place, but I have other plans for later.